values.yaml

```diff
@@ -76,35 +76,35 @@ allowPodLaunching: true
 airflowVersion: 1.10.10
 
 # Default airflow repository
-defaultAirflowRepository: astronomerinc/ap-airflow
+defaultAirflowRepository: quay.io/astronomer/ap-airflow
 
 # Default airflow tag to deploy
 defaultAirflowTag: 1.10.10-buster
 
 # Astronomer Airflow images
 images:
   airflow:
-    repository: astronomerinc/ap-airflow
+    repository: quay.io/astronomer/ap-airflow
     tag: ~
     pullPolicy: IfNotPresent
   flower:
-    repository: astronomerinc/ap-airflow
+    repository: quay.io/astronomer/ap-airflow
```

Comment:
@ashb Is this right? Flower uses ap-airflow container?

Reply:
It's right. Flower is a subcommand on airflow.